### ISSUE TITLE
Distinguish between keyboard focus and clickablity on pagination bullets

### ISSUE THREAD

**Issue by ChTopp** [nolimits4web/swiper#6251]
### Check that this is really a bug

- [X] I confirm

### Reproduction link

https://codepen.io/christinatopp/pen/BaVqPVm

### Bug description

For accessibility purposes, we use the a11y- and pagination module. Our users should be able to navigate our site and also the swiper just by using the tab key. We want them to tab through all pictures but not be able to focus the pagination.

### Expected Behavior

Pagination should be clickable but not focusable (with tab key)

### Actual Behavior

Users can tab through all pictures (with tab key). Afterwards they have to tab through all pagination bullets which is unnecessary and senseless in regards of accessibility.

### Swiper version

8.4.4

### Platform/Target and Browser Versions

 macOS, Windows, Chrome, Firefox, Safari

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [X] Make sure this is a Swiper issue and not a framework-specific issue

### Would you like to open a PR for this bug?

- [X] I'm willing to open a PR

**Comment by nolimits4web** [comment]
Issue is closed because of outdated/irrelevant/not actual/needed more information/inactivity.

If this issue is still actual and reproducible for latest version of Swiper, please create new issue and fill the issue template correctly:
* Clearly describe the issue including steps to reproduce when it is a bug.
* Make sure you fill in the earliest version that you know has the issue.
* Provide live link or JSFiddle/Codepen or website with issue
